lib/components/steps/TemplateStep.tsx

```diff
@@ -0,0 +1,308 @@
+        {
+            "id": "32845",
+            // "group": 'Origins',
+            "label": "SmallSpeciesStrainTyp",
+            "content": [
+                {
+                    "title": "Name",
+                    "value": "SmallSpeciesStrainTyp"
+                },
+                {
+                    "title": "Variable Name",
+                    "value": "Subject"
+                },
+                {
+                    "title": "Title",
+                    "value": "Unique identification of each mouse ID"
+                }
+            ]
+        },
+        {
+            "id": "47428",
+            // "group": 'Origins',
+            "label": "StudySpeciesTyp",
+            "content": [
+                {
+                    "title": "Name",
+                    "value": "StudySpeciesTyp"
+                },
+                {
+                    "title": "Variable Name",
+                    "value": "Subject"
+                },
+                {
+                    "title": "Title",
+                    "value": "Unique identification of each mouse ID"
+                }
+            ]
+        },
+        {
+            "id": "12822",
+            // "group": 'Origins',
+            "label": "Weight",
+            "content": [
+                {
+                    "title": "Name",
+                    "value": "Weight"
+                },
+                {
+                    "title": "Variable Name",
+                    "value": "Subject"
+                },
+                {
+                    "title": "Title",
+                    "value": "Unique identification of each mouse ID"
+                }
+            ]
+        },
+        {
+            "id": "1798",
+            // "group": 'Origins',
+            "label": "AgeVal",
+            "content": [
+                {
+                    "title": "Name",
+                    "value": "AgeVal"
+                },
+                {
+                    "title": "Variable Name",
+                    "value": "Subject"
+                },
+                {
+                    "title": "Title",
+                    "value": "Unique identification of each mouse ID"
+                }
+            ]
+        },
+    ];
+
+    return (
+        <ModalLayout>
+            <ModalHeightWrapper height="15rem">
+                <Box p={1.5} display="flex" flexDirection="column" gap={6}>
+                    <Stack>
+                        <Typography variant='h6'>Create template</Typography>
+                        <Typography sx={{ color: gray600, fontSize: '0.875rem', mt: "0.25rem" }}>Generate a template with CDEs or data dictionary fields before data collection for accurate mapping. Start by selecting CDEs to create template with.</Typography>
+                    </Stack>
+
+                    <Stack spacing={3}>
+                        <Box mt={6} py={1.5} sx={{ borderBottom: `1px solid ${gray100}` }}>
+                            <Breadcrumbs
```

Comment:
@Aiga115 I don't see the need for using a breadcrumb here, as in the prototype its not clickable or linking to anywhere.